packages/server/src/index.ts

```diff
@@ -1460,6 +1534,28 @@ export class App {
         let toDate
         if (endDate) toDate = setDateToStartOrEndOfDay(endDate, 'end')
 
+        if (feedback) {
+            const messages = await this.AppDataSource.getRepository(ChatMessage)
+                .createQueryBuilder('chat_message')
+                .leftJoinAndMapOne('chat_message.feedback', ChatMessageFeedback, 'feedback', 'feedback.messageId = chat_message.id')
+                .where('chat_message.chatflowid = :chatflowid', { chatflowid })
+                .andWhere(chatType ? 'chat_message.chatType = :chatType' : 'TRUE', { chatType })
```

Comment:
question: what is the `TRUE` here?

Reply:
If a value doesn't exist, `TRUE` will return the value as if there wasn't a filter. Anyway, I've updated how the query is built for optional filters so it's clearer.